docs/source/advanced/advanced_gpu.rst

```diff
@@ -73,6 +73,103 @@ Sharded Training can work across all DDP variants by adding the additional ``--p
 
 Internally we re-initialize your optimizers and shard them across your machines and processes. We handle all communication using PyTorch distributed, so no code changes are required.
 
+----------
+
+.. _fully-sharded:
+
+Fully Sharded Training
+^^^^^^^^^^^^^^^^^^^^^^
+
+.. note::
```

Comment:
maybe make this a warning instead of a note? I think this is not a note as it does not state anything about the current behaviour